### ISSUE TITLE
Add support for generic, server side template tags

### ISSUE THREAD

**Issue by falconair** [Glavin001/atom-beautify#200]
My current html file has EJS tags like this:

```html
<h1><%= bigVariable1 %></h1>
<h1><%= bigVariable2 %></h1>
<h1><%= bigVariable3 %></h1>
```

Beautifier seems to expect a closing tag, when it can't find it, it reflows the text as follows:

```html
<h1><%= bigVariable1 %></h1>
    <h1><%= bigVariable2 %></h1>
        <h1><%= bigVariable3 %></h1>
```

I believe a number of server side templates use such tags. It'll be so helpful to atom-beautify understood this scenario

**Comment by prettydiff** [comment]
I think this is a duplicate of issue #136

**Comment by Glavin001** [comment]
Published ERB support to v0.25.0
